src/components/Input/index.tsx

```diff
@@ -176,39 +179,70 @@ const Input = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
     return '';
   };
 
+  const displayInput = () => {
+    const attributeValueGatherer = value => {
```

Comment:
please rename to getAttr

Reply:
@viniciusaa please address this and rename the function as mentioned.